app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,7 @@
       <li><%= link_to "Contacts", welcome_contacts_path, class: "btn btn-success" %></li>
       <li><%= link_to "Photos", welcome_photo_gallery_path, class: "btn btn-success" %></li>
       <li><%= link_to "Videos", welcome_video_gallery_path, class: "btn btn-success" %></li>
+      <li><%= link_to "Pages", pages_path, class: "btn btn-success" %></li>
```

Comment:
Cool :) Keep in mind that eventually, you'll want to replace all those hard-coded links to About, Upcoming Events, Contacts, Photos, Videos, with a loop through the pages, to generate the list items.